auth/auth-router.spec.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../database/dbConfig.js');
+const Jokes = require('../jokes/jokes-model.js');
+
+describe('Authentication Testing', () => {
```

Comment:
For the testing section.  It's required to have 2 tests for each of the mvp endpoints so 2 for register and 2 for login